src/gbqr_3src_spatial/main.py

```diff
@@ -0,0 +1,91 @@
+import click
+import datetime
+from dateutil import relativedelta
+from pathlib import Path
+from types import SimpleNamespace
+
+from idmodels.gbqr import GBQRModel
+
+@click.command()
+@click.option(
+    "--today_date",
+    type=str,
+    required=True,
+    help="Date to use as effective model run date (YYYY-MM-DD)",
+)
+@click.option(
+    "--short_run",
+    is_flag=True,
+    help="Perform a short run."
+)
+def main(today_date: str, short_run: bool):
+    """Generate flu predictions from gbqr model."""
+    try:
+        today_date = datetime.date.fromisoformat(today_date)
+    except (TypeError, ValueError):  # if today_date is None or a bad format
+        today_date = datetime.date.today()
+    reference_date = today_date + relativedelta.relativedelta(weekday=5)
+
+    model_config = SimpleNamespace(
+        model_class = "gbqr",
+        model_name = "gbqr_3src_spatial",
+
+        incl_level_feats = True,
+
+        # bagging setup
+        num_bags = 100,
+        bag_frac_samples = 0.7,
+
+        # adjustments to reporting
+        reporting_adj = False,
+
+        # data sources and adjustments for reporting issues
+        sources = ["flusurvnet", "nhsn", "ilinet"],
+
+        # fit locations separately or jointly
+        fit_locations_separately = False,
+
+        # power transform applied to surveillance signals
+        power_transform = "4rt",
+
+        # directional wave features for spatial modeling
+        use_directional_waves = True,
+        wave_directions = ['N', 'NE', 'E', 'SE', 'S', 'SW', 'W', 'NW'],
+        wave_temporal_lags = [1, 2],
+        wave_max_distance_km = 1500,
+        wave_include_velocity = True,
+        wave_include_aggregate = True
+    )
+    
+    run_config = SimpleNamespace(
+        disease="flu",
+        ref_date=reference_date,
+        output_root=Path("../../model-output"),
+        artifact_store_root=None,
+        save_feat_importance=False,
+        max_horizon=4,
+        locations=["01", "02", "04", "05", "06", "08", "09", "10", "11",
```

Comment:
"US" not present in the spatial model because it can only do one geo-type at a time.